happiness-backend/api/user.py

```diff
@@ -213,18 +218,34 @@ def change_user_info(req, headers):
         except Exception as e:
             print(e)
             return failure_response('Invalid password key.', 400)
+    elif data_type == "key_recovery_phrase":
+        # Adds/Changes key recovery phrase to prevent loss of encrypted data on password reset.
+        try:
+            current_user.add_key_recovery(req.get("data"), headers.get("password_key"))
```

Comment:
I think we should add all key recovery phrases as either all lowercase or all uppercase. This way if a user answers a security question in a different case it doesn't prevent them from accessing their data.